jdaviz/configs/imviz/helper.py

```diff
@@ -195,6 +199,193 @@ def link_data(self, **kwargs):
         """
         link_image_data(self.app, **kwargs)
 
+    def load_regions_from_file(self, region_file, region_format='ds9', max_num_regions=20,
+                               **kwargs):
+        """Load regions defined in the given file.
+        See :ref:`regions:regions_io` for supported file formats.
+        See :meth:`load_regions` for how regions are loaded.
+
+        Parameters
+        ----------
+        region_file : str
+            Path to region file.
+
+        region_format : {'crtf', 'ds9', 'fits'}
+            See :meth:`regions.Regions.get_formats`.
+
+        max_num_regions : int or `None`
+            Maximum number of regions to read from the file, starting
+            from top of the file. Default is first 20 regions that
+            can be successfully loaded. If `None`, it will load everything.
+
+        kwargs : dict
+            See :meth:`load_regions`.
+
+        Returns
+        -------
+        bad_regions : list of (obj, str)
+            See :meth:`load_regions`.
+
+        """
+        raw_regs = Regions.read(region_file, format=region_format)
+        return self.load_regions(raw_regs, max_num_regions=max_num_regions, **kwargs)
+
+    def load_regions(self, regions, max_num_regions=None, refdata_label=None, **kwargs):
+        """Load given region(s) into the viewer.
+        WCS-to-pixel translation and mask creation, if needed, is relative
+        to the image defined by ``refdata_label``. Meanwhile, the rest of
+        the Subset operations are based on reference image as defined by Glue.
```

Comment:
I worry that this distinction is too technical for a user.  What other subset operations rely on the reference image instead of `refdata_label` and what are the consquences?

Reply:
I think it is okay to be a little technical in the API docstring.

I cannot tell you what are the implications because I tried to think of the scenarios and my head started to hurt. It is always murky here because Subsets are global but the linked coordinates transformation depends on what is the reference data in a particular viewer (which can be anything because Imviz is so flexible).

Ideas to make this implementation less confusing while maintaining flexibility is welcome. Otherwise, this is good enough until I get bug reports from Ori?